tests/integration/test_database_hms/test.py

```diff
@@ -341,3 +341,116 @@ def record(key):
 
     assert 'aaa\naaa\naaa' == node.query(f"SELECT symbol FROM {CATALOG_NAME}.`{namespace}.{table_name}`").strip()
     assert 'bbb\nbbb\nbbb' == node.query(f"SELECT symbol FROM {CATALOG_NAME}.`{namespace}.{table_name_2}`").strip()
+
+def test_hive_catalog_url_parsing(started_cluster):
```

Comment:
I think the other tests are unnecessary, just leave this one.

Reply:
@scanhex12  I have removed all the other functions that test general hms catalog functionality and retained the test which just tests the url input sanitization